geoportailv3/static/less/layout.less

```diff
@@ -90,3 +90,66 @@ body {
     bottom: 0;
   }
 }
+
+.ol-control button {
+    width:41px;
+    height:41px;
+}
+
```

Comment:
Can't this be grouped with the code below ?
Also for consistency add a space after colon.